tools/crm_mon.c

```diff
@@ -671,6 +671,21 @@ static GOptionEntry deprecated_entries[] = {
 };
 /* *INDENT-ON* */
 
+static void
+blank_screen(void)
+{
+#if CURSES_ENABLED
+    int lpc = 0;
+
+    for (lpc = 0; lpc < LINES; lpc++) {
+        move(lpc, 0);
+        clrtoeol();
+    }
+    move(0, 0);
+    refresh();
+#endif
+}
+
```

Comment:
This and detect_user_input() should be crm_mon_curses.c but we don't need to do that with this PR